### ISSUE TITLE
[Fabric] Radiant gear not working [Solved]

### ISSUE THREAD

**Issue by SylisMC** [illusivesoulworks/radiantgear#9]
**Solution..**  Trinkets lighting falls under the "Entity" category in LambDynamicLights as opposed to "First Person" for hand held items

I made a data pack to add a lantern trinket slot but it doesn't inherit the dynamic lights effect. from reading the [Radiant Gear](https://www.curseforge.com/minecraft/mc-mods/radiant-gear-fabric) I got the impression it was intended to be automatically compatible, Am I missing something? or is the mod not working as intended?

fabric-api-0.70.0+1.19.2.jar
lambdynamiclights-2.2.0+1.19.2.jar
radiantgear-fabric-2.0.0+1.19.2.jar

![2023-01-02_19 49 41](https://user-images.githubusercontent.com/19602662/210287928-9e4083ec-d6c5-4892-b9fd-9df27aaa46c7.png)
![2023-01-02_19 49 46](https://user-images.githubusercontent.com/19602662/210287931-4b4d8fdd-d901-4005-afa6-e7922da28966.png)
![2023-01-02_19 49 48](https://user-images.githubusercontent.com/19602662/210287933-2b4ee44a-c7b0-4fe6-92e3-7d26805ff44a.png)

**Comment by bellyrose** [comment]
Sorry to necro, would you be able to share the datapack? I've tried figuring out how to do it myself but can't for the life of me.

**Comment by SylisMC** [comment]
> Sorry to necro, would you be able to share the datapack? I've tried figuring out how to do it myself but can't for the life of me.

Sorry for the late response. I wasn't actively playing Minecraft at the time so I wasn't watching GitHub

Here is a copy i pulled from an old harddrive. I made that datapack so long ago, I cant remember if I ever got the lantern to render on the players hip. (It doesn't in that version of the pack) But it does let you equip them to the belt slot and it does make light render dynamically around the player as if you were holding the lantern, 

I just tested on fabric 1.20.1 Using the following

- LambDynamicLights 2.3.2
- Radiant Gear - 2.1.0
- BSL Shaders - 8.2.07.1

Make sure that Entity lighting is on 

[[1.20.X] Lantern Trinket.zip](https://github.com/illusivesoulworks/radiantgear/files/13851553/1.20.X.Lantern.Trinket.zip)
